cli/gcp.go

```diff
@@ -136,6 +136,10 @@ func newGCPGKECmd() cli.Command {
 				Name:  "name, n",
 				Usage: "Name of the cluter",
 			},
+			cli.StringFlag{
+				Name: "namespace, ns",
```

Comment:
Can you change to `namespaces` since it can specify more than one namespace, please 🙏 

```suggestion
				Name: "namespaces, ns",
```